conda_build_all/artefact_destination.py

```diff
@@ -116,5 +116,6 @@ def make_available(self, meta, built_dist_path, just_built):
         elif not just_built:
             # The distribution already existed, but not under the target owner.
             if 'http://' in built_dist_path or 'https://' in built_dist_path:
-                raise NotImplementedError('cross owner copying not yet implemented.')
-
+                source_owner = built_dist_path.rsplit('/', 3)[-3]
```

Comment:
This way of getting at the source of the distribution seems hacky, but I couldn't come up with anything better.